src/transformers/models/layoutlmv2/tokenization_layoutlmv2_fast.py

```diff
@@ -581,16 +581,17 @@ def _batch_encode_plus(
                 else:
                     original_index = batch_index
                 labels_example = []
-                for id, offset, word_id in zip(
+                cur_word_id = -1
```

Comment:
Hi, what does `cur_word_id` mean? We'd like to use variable names that are easy to understand.

Reply:
It mean "current_word_id", @NielsRogge, I can update the name.